sirepo/uri_router.py

```diff
@@ -169,6 +169,10 @@ class _HTTPRequest(web.RequestHandler):
         async def _route(self):
             p = sirepo.uri.decode_to_str(self.request.path)
             e, r, k = _path_to_route(p[1:])
+            pkdlog(
+                "start path={}",
+                p,
+            )
```

Comment:
I think this is too early to log uid. Websockets only log uid at end of request and http already does this as well.